modules/oe_content_sub_entity/src/Event/SubEntityEvent.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_sub_entity\Event;
+
+use Drupal\oe_content_sub_entity\Entity\SubEntityInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event is used to form sub-entity type labels individually for each bundle.
+ */
+final class SubEntityEvent extends Event {
```

Comment:
We need to implement two separate classes `final SubEntityEvents` and `SubEntityLabelInformationEvent extends Event`